src/views/forms/AccountManagementFormsNew/WithdrawForm/TokenSelectMenu.tsx

```diff
@@ -0,0 +1,116 @@
+import { Asset } from '@skip-go/client';
```

Comment:
TODO: delete this. we actually don't need it 🤦 since we're going UDSC only withdrawals